Automattic/jetpack
# (wip) Extensions: organize blocks adding new categories

Fixes #14598

#### Changes proposed in this Pull Request:

This PR reorganizes the block categories, adding/removing them.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
Basically it changes the category for all blocks, but it doesn't affect the functionality of the blocks.

#### Testing instructions:

* Apply these changes
* Add/edit a post/page

You should be able to blocks organized by categories, according to the table described above.

![image](https://user-images.githubusercontent.com/77539/74766719-600ea080-5264-11ea-877c-7a660daf81ec.png)
